style.css

```diff
@@ -0,0 +1,84 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+:root {
+  --primary: #6070ff;
+  --secondary: #7f8cff;
+  --neutral: #172b4d;
+  --neutral-icons: #505f79;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 1.8rem 1.5rem;
+  width: 100%;
+}
+
+.navbar .logo {
+  color: var(--primary);
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: bold;
+}
+
+.navbar .fa-bars {
+  color: var(--primary);
+  font-size: 16px;
+}
+
+.main {
+  background: url(img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
Please you must stick to the design as much as possible (e.g. font, colors, images, margins) using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Figma design:
![figma](https://user-images.githubusercontent.com/10332499/148346104-2b79290d-2bae-48fb-a889-deb7a97e1885.PNG)

 ### Your design:
![your-design](https://user-images.githubusercontent.com/10332499/148346048-01bc5fbd-3011-4711-9be8-9533e69d5855.PNG)